d3d9ex/Settings.h

```diff
@@ -87,5 +87,15 @@ SETTING(bool, BoolValue, Borderless, Borderless, false, 0);
 SETTING(bool, BoolValue, ForceWindowedMode, Borderless, false, 0);
 SETTING(bool, BoolValue, AllWindows, Borderless, false, 0);
 
+SETTING(s32, LongValue, LogLevel, Log, 2,
+	L"# LogLevel\n"
+	L"#\n"
+	L"#  0 = trace\n"
+	L"#  1 = debug\n"
+	L"#  2 = info (default)\n"
+	L"#  3 = warning\n"
+	L"#  4 = error\n"
+	L"#  5 = critical"
+);
 

```

Comment:
Should we add a setting to autoflush the log on each line written? I think that if the game crashes we will lose the information in the buffer

Reply:
Added LogFlush setting and flush on error level message:
https://github.com/rebtd7/FF13Fix/pull/64/commits/409d2234655a295c60eeabe150a0809a06b1f571#diff-704259efd3a59f930dc54e02e49da624cac33046cc647e9b1b51865d3ca8399eR55